### PR TITLE
Refs #12556 - Remove job model

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by beav** [comment]
@jlsherrill how is this related to http://projects.theforeman.org/issues/12556 ? I only see mention of system stuff in there, not sure how this relates but I'm not all that familiar with host unification

**Comment by jlsherrill** [comment]
@beav It references System, so instead of modifying the class to not use system, i'm just removing it.  As removing system is a very large task, i'm trying to open PRs in smaller chunks for easier review

File: db/migrate/20160605160933_remove_jobs.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class RemoveJobs < ActiveRecord::Migration
+  def up
+    drop_table :katello_job_tasks
+    drop_table :katello_jobs
```
Comment (beav) [inline db/migrate/20160605160933_remove_jobs.rb]: (╯°□°)╯︵ ┻━┻

**Comment by beav** [comment]
This all looks sane to me. I'm not 100% familiar with the code being removed, but it's a moot point if its being taken out :cat2: 

ACK

**Pull request merged**